crates/madara/src/config/global_config.rs

```diff
@@ -0,0 +1,321 @@
+#![allow(unused)]
+use anyhow::{Context, Error};
+use clap::builder::Str;
+use cliclack::Input;
+use figment::{
+    providers::{Format, Toml},
+    Figment,
+};
+use hex;
+use madara_cli_common::{logger, Prompt};
+use secp256k1::{PublicKey, Secp256k1, SecretKey};
+use sha3::{Digest, Keccak256};
+use std::fs;
+
+use serde::{Deserialize, Serialize};
+
+use crate::constants::DEFAULT_LOCAL_CONFIG_FILE;
+
+use super::constants::{
+    ANVIL_CHAIN_ID, ANVIL_RPC_URL, DEFAULT_ATLANTIC_URL, DEFAULT_MOCK_VERIFIER_ADDRESS,
+    MADARA_APP_CHAIN_ID, MADARA_BLOCK_TIME, MADARA_CHAIN_NAME, MADARA_LATEST_PROTOCOL_VERSION,
+    MADARA_NATIVE_FEE_TOKEN_ADDRESS, MADARA_PARENT_FEE_TOKEN_ADDRESS,
+    MADARA_PENDING_BLOCK_UPDATE_TIME,
+};
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct EthWallet {
+    pub eth_priv_key: String,
+    pub l1_deployer_address: String,
+    pub l1_operator_address: String,
+    pub l1_multisig_address: String,
+}
+
+impl Default for EthWallet {
+    fn default() -> Self {
+        Self::new(
+            Self::get_priv_key(0),
```

Comment:
Maybe `get_priv_key` could return a tuple <privkey, pubkey> (and rename it to `get_keys`)

Reply:
Done!